apps/api-frontend/src/app/pages/search-result.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { environment } from '../../environments/environment';
+import { ParsedArticleParagraphStandalone } from '@foodmedicine/interfaces';
+import ClimbingBoxLoader from 'react-spinners/ClimbingBoxLoader';
+import { useLocation, useHistory } from 'react-router-dom';
+import { onSearch } from './onsearch';
+import './search-result.css';
+import { SearchBar } from '@foodmedicine/components';
+
+function SingleResult(props: {
+  paragraph: ParsedArticleParagraphStandalone;
+  key: string;
+}) {
+  return (
+    <div
+      className="single-result-container"
+      onClick={() => window.open(props.paragraph.head.xmlFullTextDownloadLink)}
+    >
+      <h4 aria-label="paper's title">{props.paragraph.head.title}</h4>
+      <p aria-label="correlated paragraph">...{props.paragraph.body}...</p>
```

Comment:
You can remove both of these aria labels. You usually should not have an aria label on text elements :)